proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -90,8 +113,8 @@ message MsgEditValidator {
   // REF: #2373
   string commission_rate = 3
       [(cosmos_proto.scalar) = "cosmos.Dec", (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec"];
-  string min_self_delegation = 4
-      [(cosmos_proto.scalar) = "cosmos.Int", (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int"];
+
+  reserved 4;
```

Comment:
Based on past experience pretty sure we can't do this. @amaurym ☝️

Reply:
ah okay, I was following proto standards and not sure why we cant do this is documented anywhere?

Reply:
It breaks protogen for clients I think. I think @amaurym can comment more

Reply:
I will migrate to deprecated as this seems to be supported

Reply:
Yeah it's documented in https://github.com/cosmos/cosmos-sdk/blob/main/docs/architecture/adr-044-protobuf-updates-guidelines.md, there's also a linked issue